routers/web/repo/view.go

```diff
@@ -361,6 +362,14 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 		if workFlowErr != nil {
 			ctx.Data["FileError"] = ctx.Locale.Tr("actions.runs.invalid_workflow_helper", workFlowErr.Error())
 		}
+	} else if util.SliceContains([]string{"CODEOWNERS", "docs/CODEOWNERS", ".gitea/CODEOWNERS"}, ctx.Repo.TreePath) {
```

Comment:
Why it's `util.SliceContains` but not equal?

Reply:
It means `ctx.Repo.TreePath=="CODEOWNERS" || ctx.Repo.TreePath==".gitea/CODEOWNERS" || ....`